src/sidebar/util/test/version-data-test.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+const VersionData = require('../version-data');
+
+describe('sidebar.util.version-data', () => {
```

Comment:
The mix of `describe` styles in our codebase causes some confusion. But what I would suggest is the canonical style is:

- For modules that just export one class/function etc, just use the name of that thing (`describe('VersionData')`)
- For modules that export a collection of things, use a require-style path as if you were executing a command in `src/`. eg. `describe('sidebar/util/version-data')` containing individual `describe` blocks for each exported entity.

Reply:
Yep, there's quite a mix so I tried to choose the right one. Chose wrong!